service/kvmconfig/resource_router.go

```diff
@@ -6,19 +6,19 @@ import (
 	"github.com/giantswarm/microerror"
 	"github.com/giantswarm/operatorkit/framework"
 
-	"github.com/giantswarm/kvm-operator/service/keyv3"
+	"github.com/giantswarm/kvm-operator/service/kvmconfig/v3/key"
 )
 
 // newResourceRouter determines which resources are enabled based upon the
 // version in the version bundle.
 func newResourceRouter(resources map[string][]framework.Resource) func(ctx context.Context, obj interface{}) ([]framework.Resource, error) {
 	return func(ctx context.Context, obj interface{}) ([]framework.Resource, error) {
-		customObject, err := keyv3.ToCustomObject(obj)
+		customObject, err := key.ToCustomObject(obj)
```

Comment:
So the resource router should be versioned as well? How should that work? It is a bit odd to only use v3 here.

Reply:
It is and was odd. My idea is to move `ToCustomObject` somewhere, it doesn't fit key package anyway. Maybe top level i.e. `kvmconfig`?